README.md

```diff
@@ -126,8 +126,33 @@ xh put httpbin.org/put id:=49 age:=25 | less
 xh -d httpbin.org/json -o res.json
 ```
 
-## Syntaxes and themes used
-- [Sublime-HTTP](https://github.com/samsalisbury/Sublime-HTTP)
-- [json-kv](https://github.com/aurule/json-kv)
-- [Sublime Packages](https://github.com/sublimehq/Packages/tree/fa6b8629c95041bf262d4c1dab95c456a0530122)
-- [ansi-dark theme](https://github.com/sharkdp/bat/blob/master/assets/themes/ansi-dark.tmTheme)
+## xh vs HTTPie
+
+### Why should I use xh over HTTPie
+
+- xh can be 10+ times as fast on requests to localhost, or even 50+ times if you disable coloring.
+- It releases as a single statically linked binary, so it's easier to install and carry around.
+- It supports HTTP/2 (without a plugin).
+- It doesn't use the system's installed TLS library.
+- It has a builtin translator to curl like https://curl2httpie.online/.
+- _--help's_ output fits on your screen. (help gives longer output, like HTTPie's.)
+
+### Why shouldn't I use xh over HTTPie
+
+- It doesn't have all of HTTPie's features.
+- You can't disable HTTP/2 (https://github.com/ducaale/xh/issues/68), and large downloads over HTTP/2
+  perform badly (https://github.com/ducaale/xh/issues/49).
+- It may never have anything like HTTPie's plugin system.
+- General immaturity. HTTPie has seen a lot of real-world usage, that's hard to beat.
+- Worse documentation. HTTPie's docs are very impressive, and you can even try out commands right on the
+  webpage using termible.
+
+### Differences:
+
+- Headers are sent and printed in lowercase (because of the libraries used).

```

Comment:
I'd revise these some more, to make them easier to skim:
```suggestion
## xh vs HTTPie

### Advantages over HTTPie

- Improved startup speed. `xh` may run short requests many times as fast.
- Available as a single statically linked binary that's easy to install and carry around.
- HTTP/2 support.
- Builtin translation to curl commands with the `--curl` flag (similar to https://curl2httpie.online/).
- Short, cheatsheet-style output from `--help`. (For longer output, pass `help`.)

### Disadvantages

- Not all of HTTPie's features are implemented. ([#4](https://github.com/ducaale/xh/issues/4))
- HTTP/2 cannot be disabled. ([#68](https://github.com/ducaale/xh/issues/68))
- Large downloads over HTTP/2 are currently slow. ([#49](https://github.com/ducaale/xh/issues/49))
- No plugin system.
- General immaturity. HTTPie is old and well-tested.
- Worse documentation.

### Other differences

- `rustls` is used instead of the system's TLS library.
- Headers are sent and printed in lowercase.
```
I've also moved the `rustls` item, because I don't think it's a strict advantage—the system library is easier to update and gives consistency with other tools.

(Not sure about all the changes I made, feel free to revise and revert)